Cogs.Publishers/DotSchemaPublisher.cs

```diff
@@ -60,7 +64,17 @@ public void Publish(CogsModel model)
             // TODO: if Overwrite is false and Directory.Exists(TargetDirectory)) throw an error and exit
 
             Directory.CreateDirectory(TargetDirectory);
-
+            if (File.Exists("dot.exe")) { DotLocation = Path.GetFullPath("dot.exe"); }
```

Comment:
on linux and mac, it won't be dot.exe. Lets leave the option to specify the dot path.
If it isn't specified, we can assume it is in the PATH
If it isn't in the path, and it isn't specified, then we can issue a cogs error saying it can't be found